console/src/components/Widget/Widget.js

```diff
@@ -71,14 +71,16 @@ const WidgetBase = kind({
 				</Row>
 			);
 			default: return (
-				<Layout align="center center" orientation="vertical" {...rest}>
```

Comment:
I think this was intended to be overridable with the orientation and alignment before the `...rest`.